package.json

```diff
@@ -25,7 +24,8 @@
     "local:git-checkout": "./scripts/local/local-git-checkout.sh",
     "prettier": "prettier '{.github,docs,web/profiles/custom,web/modules/custom,web/themes/contrib/atomic,web/themes/custom}/**/*.{json,md,js,html,scss,html}' --ignore-unknown --list-different",
     "setup": "./scripts/local/setup.sh",
-    "test": "./scripts/local/test.sh"
+    "test": "./scripts/local/test.sh",
+    "prepare": "husky"
```

Comment:
Should this one be:

```json
 "prepare": "husky install"
```

I believe with that it'll install the hooks for husky that you've set up.